Documentation/data-collection.md

```diff
@@ -225,6 +225,17 @@ data:
     # (workloads-team, @openshift/openshift-group-b) cluster_master_schedulable reports whether mastersSchedulable=true in
     # the scheduler operator. This value is a boolean 0|1
     - '{__name__="cluster_master_schedulable"}'
+    # (codeready workspaces) The number of workspaces with a given status STARTING|STOPPED|RUNNING|STOPPING. Type 'gauge'.
+    - '{__name__="che_workspace_status"}'
+    # (codeready workspaces) The number of started workspaces. Type 'counter'.
```

Comment:
```suggestion
    # (codeready workspaces, @ibuziuk) The number of started workspaces. Type 'counter'.
```

Reply:
I sadly cannot accept these changes, only the person that opens the PR, do you have the right to push to the PR?

Reply:
Unfortunately  not, but I ask Ilya to give access for me

Reply:
@lilic @vparfonov thanks, folks
I have committed the suggestions and granted access to Vitalli. This week I have very limited access to the internet, but will try to follow up on this PR as much as possible.